src/camvidlog2/yoloe/ai.py

```
@@ -0,0 +1,214 @@
+from pathlib import Path
+from typing import Iterable, Iterator, List, Optional, Sequence
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Iterable`, `Iterator`, `Sequence`

Import from `collections.abc`

(UP035)